sos/plugins/kernel.py

```diff
@@ -34,7 +34,8 @@ def setup(self):
 
         try:
             modules = os.listdir(self.sys_module)
-            self.add_cmd_output("modinfo " + " ".join(modules))
+            self.add_cmd_output("modinfo " + " ".join(modules),
+                                suggest_filename="modeinfo")
```

Comment:
'modeinfo'?? ;-)

Since you're going to need to respin this anyway, let's go with "_ALL_MODULES" or some other uniquifier that the other maintainers agree on.